.ddev/commands/web/openmage-install

```diff
@@ -87,11 +90,15 @@ if [[ $INSTALL_SAMPLE_DATA =~ ^(yes|y) ]]; then
   echo "Copying Sample Data into the OpenMage directory..."
   cp -r magento-sample-data-1.9.2.4/* "${ROOT}/"
 
+  echo "Clear var/cache ..."
```

Comment:
Clearing /var/cache directory... 

Just for keeping the same format and and punctuation.